app/models/physical_server.rb

```diff
@@ -67,16 +67,6 @@ def is_refreshable_now_error_message
     refreshable_status[:message]
   end
 
-  def is_available?(_address)
-    # TODO: (walteraa) remove bypass
-    true
-  end
-
-  def smart?
```

Comment:
This doesn't look related to the availability mixin work, but it also doesn't appear to be used anywhere else